server/http.go

```diff
@@ -394,7 +394,19 @@ func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	meetingID := zoomUser.Pmi
+	client, _, err := p.getActiveClient(user)
+	if err != nil {
+		p.API.LogWarn("Error getting the client", "err", err)
+		return
+	}
+
+	meeting, err := client.CreateMeeting(user.Email)
+	if err != nil {
+		p.API.LogWarn("Error creating the meeting", "err", err)
+		return
+	}
```

Comment:
Does the Mattermost user's email have to match the Zoom user's email? If so, can we communicate that to the user here?

Reply:
No, there is no need on OAuth. I will make some changes to properly handle this.